components/ShippingRates/index.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <UCard :ui="{ body: { padding: '' } }">
+    <template #header>
+      <div class="flex flex-row justify-between items-center">
+        <div class="flex justify-start items-center gap-[8px]">
+          <h3 class="font-bold font-mono">
+            Shipping Options
+          </h3>
+          <slot name="header" />
+        </div>
+        <UButton
+          v-if="!shouldHideViewButtonOnViewMode"
+          :icon="buttonConfig.icon"
+          :label="buttonConfig.text"
+          :loading="isLoading"
+          @click="buttonConfig.action"
+        />
+      </div>
+    </template>
+
+    <UTable
+      v-if="isEditMode"
+      :columns="[
+        { key: 'index' },
+        { key: 'name', label: 'Name' },
+        { key: 'price', label: 'Price (USD)' },
+      ]"
+      :rows="shippingRatesTableRows"
+    >
+      <template #name-data="{ row }">
+        <div class="flex flex-col gap-[8px] items-start">
+          <span class="text-center">en: {{ row.name.en }}</span>
+          <span class="text-center">zh: {{ row.name.zh }}</span>
+        </div>
+      </template>
+      <template #price-data="{ row }">
+        <span class="text-center">{{ row.price }}</span>
+      </template>
+    </UTable>
+
+    <div v-if="isViewMode" class="px-[24px] py-[12px]">
```

Comment:
should this be v-else-if?

Reply:
[💄 Display checkbox section if is not editMode](https://github.com/likecoin/nft-book-press/pull/88/commits/688511d89e42940c3aeda8c1d0454b5bf490d008)